src/app/components/planner-list/planner-list.component.ts

```diff
@@ -181,7 +181,7 @@ export class PlannerListComponent implements OnInit, AfterViewChecked, OnDestroy
     this.loggedIn = this.auth.isLoggedIn();
 
     // Cookie for datatableColumn config
-    if(!this.cookieService.getCookie(datatableColumn.length).status) {
+    if(this.cookieService.getCookie(datatableColumn.length).status) {
```

Comment:
Why you change this condition?

Reply:
that was for testing at that time. I missed to undo the change